src/index.ts

```diff
@@ -178,6 +178,17 @@ async function processFile(
       promotionTargetRegexp || null,
       logger,
     );
+
+    // Assumption: The filepath is of the format `teams/{team-name}` or `teams/{team-name}/migrations` (for db migrations)
```

Comment:
That said, this is running once per file even though it's a pure function of the top-level inputs to the action — it should be moved to the top level too rather than spit out the same outputs over and over.